src/redux/API.js

```diff
@@ -0,0 +1,59 @@
+import { addBook, removeBook, loadBooks } from './books/books';
+
+//! For get app id
+// const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps';
+// useEffect(() => {
+//   const fetchData = async () => {
+//     const response = await fetch(URL, {
+//       method: 'POST',
+//       headers: { 'Content-Type': 'application/json' },
+//
+//     });
+//     const data = await response.text();
+//     console.log(data);
+//   };
+//   fetchData();
+// }, []);
+
+const BOOK_STORE_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/naN2smxjCVu4fr6NtvX5/books';
+//! Get data
+// const getBooks = async () => {
+//   const response = await fetch(BOOK_STORE_URL);
+//   const data = await response.json();
+//   return data;
+// };
```

Comment:
- [OPTIONAL] It would be better if you remove unnecessary comments to have a readable and clean code. 😉